mantidproject/mantid
# Scripts for fitting benchmarks (and a system test)

Adds scripts that produce the comparison and tables used for the concept page on fitting  minimizers (#17603). This also adds a system test that runs the scripts to evaluate all the tests problems that we currently have with all the minimizers (except FABADA). 

Note this PR has many files because of the test files added under `Testing/Data/SystemTest/FittingTestProblems`

**To test:**


- Code review
- Check that the systemtest works

```
./systemtest  -R FittingBenchmarks
```

(or if you don't want a flood of warnings:

```
./systemtest  -l error -R FittingBenchmarks
```
(This not needed for testing this ticket: The files for end users to be able to run the benchmark scripts without needing the system tests infrastructure are here (at ISIS): \olympic\Babylon5\Public\fedemp\fitting\FittingTestData.zip))

Fixes #15952.
Also fixes #13782.

Release notes were updated.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
